### PR TITLE
User Onboarding UI

### PR DESCRIPTION
**Pull request by ultraviolet10**
- Redid connect modal flow, now has options to `Connect Wallet`, `1-Click Trading` (pre-approve tokens), `Email Input` (todo)
- Redid UI for approve tokens modal.
- Changed `Sepolia` icon.
- Changed some styling.

reference: https://www.figma.com/file/MSqK7yvT6FfYDbnk7ICtOJ/T3?node-id=231%3A4116&mode=dev

`todo`: import 2 assets from figma for showing completion of step(s) - facing some issue with export.

### CODE REVIEW

File: src/components/ApproveTokenInput/ApproveTokenInput.js [hunk 1]
```diff
@@ -80,37 +80,35 @@ export default function ApproveTokenInput(props) {
...
+          <button onClick={onMaxClick} className="Approve-token-input-max">
+            MAX
+          </button>
         </div>
+
+        <Button
+          variant={isApproved ? "approved" : "await"}
+          className="w-20 h-full"
+          onClick={onApproveClick}
+          disabled={parseFloat(approveValue) === "0" || isApproving}
```
Comment (srslafazan) [inline src/components/ApproveTokenInput/ApproveTokenInput.js:103]: Looks like this should be `parseFloat(approveValue) === 0 || isApproving` as the float will never strict equal the string.

**Review by srslafazan — APPROVED** [review]
Nice use of framer motion!

I wasn't able to get the developer server running tonight, however in review the logic and template updates look correct.

One comparator issue to address before merge.

File: src/App/App.js [hunk 2]
```diff
@@ -630,46 +644,78 @@ function FullApp() {
         className="Connect-wallet-modal"
         isVisible={walletModalVisible}
         setIsVisible={setWalletModalVisible}
-        label={t`Connect Wallet`}
+        label={`Get Started`}
```
Comment (srslafazan) [inline src/App/App.js:647]: We may translate this.

File: src/App/App.js [hunk 3]
```diff
@@ -630,46 +644,78 @@ function FullApp() {
...
+          </button>
+        </div>
       </Modal>
+
       <Modal
         className="Approve-tokens-modal"
         isVisible={approvalsModalVisible}
         setIsVisible={setApprovalsModalVisible}
-        label={`Approve Tokens`}
+        label={`Enable 1-Click Trading`}
```
Comment (srslafazan) [inline src/App/App.js:708]: And here, we may add translation.

File: src/components/ApproveTokenInput/ApproveTokenInput.js [hunk 4]
```diff
@@ -80,37 +80,35 @@ export default function ApproveTokenInput(props) {
...
+          <input
+            type="number"
+            min="0"
+            placeholder="0.0"
+            className="Approve-swap-input"
+            value={approveValue}
+            onChange={onApproveValueChange}
+          />
+          <button onClick={onMaxClick} className="Approve-token-input-max">
+            MAX
```
Comment (srslafazan) [inline src/components/ApproveTokenInput/ApproveTokenInput.js:95]: Here as well, a translation may be added.

File: src/components/ApproveTokenInput/ApproveTokenInput.js [hunk 5]
```diff
@@ -80,37 +80,35 @@ export default function ApproveTokenInput(props) {
...
+          </button>
         </div>
+
+        <Button
+          variant={isApproved ? "approved" : "await"}
+          className="w-20 h-full"
+          onClick={onApproveClick}
+          disabled={parseFloat(approveValue) === "0" || isApproving}
+        >
+          {isApproved ? "Approved!" : "Approve"}
```
Comment (srslafazan) [inline src/components/ApproveTokenInput/ApproveTokenInput.js:105]: Here as well, a translation will need to be added.

File: src/components/ApproveTokenInput/ApproveTokenInput.js (hunk 1 above)
Comment (ultraviolet10) [reply src/components/ApproveTokenInput/ApproveTokenInput.js]: fixed

**Pull request merged**